source/stylesheets/_layout.scss

```diff
@@ -65,11 +64,17 @@
   .awards {
     @include span-columns(3);
     margin-bottom: $footer-height;
+    margin-top: golden-ratio($gutter, -1);
 
     .award {
       @include span-columns(1 of 3);
       @include omega(3n);
-      margin-top: golden-ratio($gutter, -1);
+      margin-bottom: golden-ratio($gutter, -1);
```

Comment:
Oh, it's basically the golden ratio power the argument (`1.618033 ^ arg`). So in the case of `arg == 0` it would be `1`, in the case of `arg == -1` it would be `(1 / 1.618033)`. So it decreases the gutter size by the **magic** golden ratio that designers love so much. :tongue: